src/libraries/System.IO.FileSystem.AccessControl/src/System.IO.FileSystem.AccessControl.csproj

```diff
@@ -88,7 +88,6 @@
              Link="Common\System\IO\FileSystem.DirectoryCreation.Windows.cs" />
   </ItemGroup>
   <ItemGroup>
-    <Reference Include="System.Buffers" Condition="'$(TargetPlatformIdentifier)' == 'windows'" />
```

Comment:
System.Buffer has been a pure facade assembly since https://github.com/dotnet/corefx/commit/b09c098f2e0c7934dac8a4c7e38a5bf249a5b955 and doesn't need to be referenced anymore expliclty. It's API is already provided by System.Runtime.